xbmc/XBDateTime.h

```diff
@@ -179,6 +179,7 @@ class CDateTime final : public IArchivable
   std::string GetAsDBTime() const;
   std::string GetAsLocalizedDate(bool longDate=false) const;
   std::string GetAsLocalizedDate(const std::string &strFormat) const;
+  std::string GetAsLocalizedDate(const std::string& strFormat, bool returnFormat) const;
```

Comment:
we try to avoid introducing new `bool` parameters and use enum classes instead. reasoning is a call like
```c
... = GetAsLocalizedDate("someFormat", ReturnFormat::CHOICE_YES);
```
is more readable from the calling side than
```c
... = GetAsLocalizedDate("someFormat", true);
```

so i would introduce a new `enum class ReturnFormat` for this purpose.
you can see a few examples here
https://github.com/xbmc/xbmc/blob/0a989c4ce6f08769427b30dd3fc3dfd9d90ac49e/xbmc/addons/AddonInstaller.h#L36-L46

Reply:
Requested change made.